src/components/webView.re

```diff
@@ -1,15 +1,22 @@
 [@bs.module "react-native"] external view : ReasonReact.reactClass = "WebView";
 
-type iOSLoadRequestEvent = {
+type navigationType = [
+  | [@bs.as "click"] `click
+  | [@bs.as "formsubmit"] `formsubmit
+  | [@bs.as "backforward"] `backforward
+  | [@bs.as "reload"] `reload
+  | [@bs.as "formresubmit"] `formresubmit
+  | [@bs.as "other"] `other
+];
+
+type iOSOnShouldStartLoadWithRequestEvent = {
   .
-  "target": int,
-  "canGoBack": bool,
-  "lockIdentifier": int,
-  "loading": bool,
+  "url": string,
   "title": string,
+  "loading": bool,
+  "canGoBack": bool,
   "canGoForward": bool,
-  "navigationType": string,
-  "url": string,
+  "navigationType": navigationType,
```

Comment:
I think it would be better to inline the poly variant `navigationType` here, then you should be able to use `[@bs.string]` if I remember correctly.

Reply:
hmm I tried bs.string and I still get the `unused` warning

Reply:
I _think_, but I might be mistaken here, that `"navigationType": [@bs.string] [ | `reload | `formresubmit ...]` should work.

But then you need to inline it, and you don't need the `[@bs.as "..."]` for each variant since the values are the same. `@bs.string` defaults to converting it to the exact string without the back tick.